portfolio/src/main/java/com/google/sps/servlets/GetCommentsServlet.java

```diff
@@ -32,9 +33,11 @@
 /** 
  * Servlet that stores and returns comments.
  */
-@WebServlet("/comments")
-public class CommentsServlet extends HttpServlet {
+@WebServlet("/get-comments")
+public class GetCommentsServlet extends HttpServlet {
 
+  // The max number of comments to send on a GET request. Is modified on POST request.
+  private int maxComments = 5;
```

Comment:
Will this expose any race conditions here? User 1 calls doPost with a max=5 and User2 calls doPost with a max=10 before the page redirect to comments.html for User1. Will User1 see 5 or 10 comments in this case.

Reply:
Oh I forgot to think about race conditions here. Yes the maxComments variable is unfortunately shared among all users, so a user requesting to see, say, 7 comments will have 7 comments displayed for all users. I think trying to fix this would be a nontrivial change to the code? So I'll open an issue for it and try to fix this in a separate branch in another pull request.

Reply:
Added a potential solution in the comments of issue #19. Ping me if you want to chat about it.

Reply:
Oh awesome, thanks!

Reply:
Resolving this, will fix in a separate PR.